classify_test.go

```diff
@@ -87,4 +87,13 @@ func TestClassify(t *testing.T) {
 			t.Errorf("Expected: fruit. Receieved: %s", res.Classifications[0].Prediction)
 		}
 	})
+
+	t.Run("Classify with preset", func(t *testing.T) {
+		_, err := co.Classify(ClassifyOptions{
+			Preset: "PUT-REAL-PRESET-HERE",
+		})
+		if err != nil {
+			t.Errorf("expected result, got error: %s", err.Error())
+		}
```

Comment:
Will this pass once the preset work is merged?

Reply:
Yeah, I'm waiting for the API side to be merged, then I will update PR with real preset and make sure it passes